src/__tests__/withAuthUser.test.js

```diff
@@ -826,6 +826,29 @@ describe('withAuthUser: rendering/redirecting', () => {
     expect(container.firstChild).toBeNull()
```

Comment:
In the above test, change description to "renders null by default when redirecting to the app".